tools/ci_build/github/azure-pipelines/nuget/templates/windowsai.yml

```diff
@@ -142,77 +142,144 @@ jobs:
-        Start-Process -FilePath $zip_tool -ArgumentList "-r $merged_nuget ." -WorkingDirectory $x64_unzipped_directory -NoNewWindow -Wait
+        Start-Process -FilePath $zip_tool -ArgumentList "-r $merged_nuget ." -WorkingDirectory $x64_nupkg_unzipped_directory -NoNewWindow -Wait
+      workingDirectory: $(Build.BinariesDirectory)\nuget-artifact-x64
+
+  - task: PowerShell@2
+    displayName: 'Bundle Symbols NuGet'
+    inputs:
+      targetType: 'inline'
+      script: |
+        Add-Type -AssemblyName "System.IO.Compression.FileSystem"
+
+        $nupkgs = (Get-ChildItem -Filter Microsoft.AI.MachineLearning*.snupkg -Recurse)
+        $x64_nuget_package_name = $nupkgs[0].Name
+        $x64_nuget_package = $nupkgs[0].FullName
+        $x64_nupkg_unzipped_directory_root = $nupkgs[0].Directory.FullName
+        $x64_nupkg_unzipped_directory = [System.IO.Path]::Combine($x64_nupkg_unzipped_directory_root, 'symbols', [System.IO.Path]::GetFileNameWithoutExtension($x64_nuget_package))
+        [System.IO.Compression.ZipFile]::ExtractToDirectory($x64_nuget_package, $x64_nupkg_unzipped_directory)
+
+        $nupkgs = (Get-ChildItem ..\nuget-artifact-x86 -Filter Microsoft.AI.MachineLearning*.snupkg -Recurse)
+        $x86_nuget_package = $nupkgs[0].FullName
+        $x86_nupkg_unzipped_directory_root = $nupkgs[0].Directory.FullName
+        $x86_nupkg_unzipped_directory = [System.IO.Path]::Combine($x86_nupkg_unzipped_directory_root, 'symbols', [System.IO.Path]::GetFileNameWithoutExtension($x86_nuget_package))
+        [System.IO.Compression.ZipFile]::ExtractToDirectory($x86_nuget_package, $x86_nupkg_unzipped_directory)
+
+        $nupkgs = (Get-ChildItem ..\nuget-artifact-arm64 -Filter Microsoft.AI.MachineLearning*.snupkg -Recurse)
+        $arm64_nuget_package = $nupkgs[0].FullName
+        $arm64_nupkg_unzipped_directory_root = $nupkgs[0].Directory.FullName
+        $arm64_nupkg_unzipped_directory = [System.IO.Path]::Combine($arm64_nupkg_unzipped_directory_root, 'symbols', [System.IO.Path]::GetFileNameWithoutExtension($arm64_nuget_package))
+        [System.IO.Compression.ZipFile]::ExtractToDirectory($arm64_nuget_package, $arm64_nupkg_unzipped_directory)
+
+        $nupkgs = (Get-ChildItem ..\nuget-artifact-arm -Filter Microsoft.AI.MachineLearning*.snupkg -Recurse)
+        $arm_nuget_package = $nupkgs[0].FullName
+        $arm_nupkg_unzipped_directory_root = $nupkgs[0].Directory.FullName
+        $arm_nupkg_unzipped_directory = [System.IO.Path]::Combine($arm_nupkg_unzipped_directory_root, 'symbols', [System.IO.Path]::GetFileNameWithoutExtension($arm_nuget_package))
+        [System.IO.Compression.ZipFile]::ExtractToDirectory($arm_nuget_package, $arm_nupkg_unzipped_directory)
+
+        $x86_runtime_path_old = [System.IO.Path]::Combine($x86_nupkg_unzipped_directory, 'runtimes', 'win-x86', 'native')
+        $x86_runtime_path_new = [System.IO.Path]::Combine($x64_nupkg_unzipped_directory, 'runtimes', 'win-x86', 'native')
```

Comment:
Essentially the x86 arm64 and arm  symbols are all being copied to respective folders in x64 nuget package folder right? That's why you use $x64_nupkg_unzipped_directory

Reply:
yep - all of the nupkgs are unzipped and then the non-x64 ones are copied into the x64 unzipped folder. It is packaged and placed into the merged dir.

same for symbols.